src/accidental.ts

```diff
@@ -65,6 +65,16 @@ export class Accidental extends Modifier {
 
   /** Arrange accidentals inside a ModifierContext. */
   static format(accidentals: Accidental[], state: ModifierContextState): void {
+    // If there are no accidentals, no need to format their positions.
+    if (!accidentals || accidentals.length === 0) return;
```

Comment:
I moved this block up here so that it returns earlier if there are no accidentals. Otherwise we are looking up font metrics for no reason.

Reply:
I noticed it, it is a good idea.